nilimamaka/SpringBoot/MultipleDBConnection/src/main/java/com/example/multipledbconnection/fileDetails/dto/DtoFDRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.example.multipledbconnection.fileDetails.dto;
+
+import lombok.Data;
+
+@Data
+public class DtoFDRequest {
```

Comment:
No require to mention name Dto as dto.ClassName.java will smell its a DTO class.
FileDetailsRequest can be used. For ease you can put FileDetailsRequestDto too.